content/en/api/integrations_aws/code_snippets/result.aws_update.sh

```diff
@@ -0,0 +1,3 @@
+{
+    "external_id": "12345678910111213"
```

Comment:
I believe this doesn't return the external id and instead just returns an empty response `{}`